platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -709,8 +726,21 @@ protected void onCreate(Bundle icicle) {
 		initializeGodot();
 	}
 
+	@Override
+	protected void onStart() {
+		super.onStart();
+
+		mView.post(new Runnable() {
+			@Override
+			public void run() {
+				mKeyboardWindow.showAtLocation(getWindow().getDecorView(), Gravity.NO_GRAVITY, 0, 0);
+			}
+		});
```

Comment:
@pouleyKetchoupp Not sure if the popup window always showing is the right approach. Is this needed for the virtual keyboard height to be properly set? 
Can't we instead show the popup window in `GodotEditText` when `HANDLER_OPEN_IME_KEYBOARD` is received, and dismiss it when `HANDLER_CLOSE_IME_KEYBOARD` is received.

Also you shouldn't need to invoke the call within a runnable here.

Reply:
From what I remember, I had problems with the keyboard not showing at all when trying to start the popup at the same time the keyboard is requested. But I can test it again, I agree it would be cleaner.

What do you mean by not invoking the call with a runnable? I might be missing something, but that's the only option I found to ensure the call is done only on View.post for the view to be ready for spawning a popup on top.

Reply:
So after testing again I can confirm the popup has to be always showing for the height to be updated properly, and for the keyboard to show up consistently. I'm not sure what the exact reason is, but I would rather spend more hours trying to change this only in case it causes a specific behavior or performance problem.

Reply:
Sounds good and thanks for checking. 

My primary concern was for VR mode which doesn't make use of that functionality, but will nonetheless include it. Although I can probably add a check when `xrMode` is VR and skip creating and initializing `GodotEditText`.